app/api/route/handler/user_handler.py

```diff
@@ -0,0 +1,69 @@
+import enum
+from datetime import date
+from typing import Optional
+from uuid import uuid4
+
+from pydantic import UUID4
+
+import api.logging
+from api.db.models.user_models import User, UserRole
+from api.route.api_context import ApiContext
+from api.route.request import BaseRequestModel
+
+logger = api.logging.get_logger(__name__)
+
+
+class RoleEnum(str, enum.Enum):
+    USER = "User"
+    ADMIN = "Admin"
+    THIRD_PARTY = "Third Party"
```

Comment:
`RoleEnum` is part of the business domain so it may need to be used by business logic classes, so it probably shouldn't live in a file that is tied to flask route handlers (since we don't want business classes depending on things like flask). Could we move this to `user_models.py`?